src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1810,7 +1810,21 @@ public function insert_posts( $items = [] ) {
 							}
 
 							if ( empty( $venue_unique_field ) || ( $lookup_venues_by_title && empty( $venue ) ) ) {
-								$venue = get_page_by_title( $event['Venue']['Venue'], 'OBJECT', Tribe__Events__Venue::POSTTYPE );
+								$venue_query = new WP_Query(
+									[
+										'post_type'      => Tribe__Events__Venue::POSTTYPE,
+										'title'          => $event['Venue']['Venue'],
+										'post_status'    => 'all',
+										'posts_per_page' => 1,
+										'no_found_rows'  => true,
+										'ignore_sticky_posts' => true,
+										'update_post_term_cache' => false,
+										'orderby'        => 'post_date ID',
```

Comment:
Multiple `orderby` should be an array.

Reply:
According to WP, when the sorting order is the same, a string can be used: https://developer.wordpress.org/reference/classes/wp_query/#order-orderby-parameters